liquidctl/driver/base.py

```diff
@@ -92,6 +92,11 @@ def set_color(self, channel, mode, colors, **kwargs):
 
         raise NotImplementedError()
 
+    def set_screen(self, mode, value, **kwargs):
```

Comment:
As mentioned when discussing the CLI, the API needs to take a "channel" name as well.

```suggestion
    def set_screen(self, channel, mode, value, **kwargs):
```

At the CLI level `channel` is probably the most appropriate name just because it matches the other possible usages, but here we can choose to instead use a name that's clearer (e.g. `screen`).  What do you think?